src/custumers/api.py

```diff
@@ -0,0 +1,80 @@
+from flask_openapi3 import APIBlueprint, Tag
+
+
+from src.custumers.models import Custumer
+from src.custumers.schemas import CustumerDeleteSchema, CustumersPostSchema
+
+custumers_api = APIBlueprint(
+     "/custumers",
+     __name__,
+     url_prefix="/api",
+     abp_tags=[Tag(name="Custumers", description="Random custumers")],
+     doc_ui=True,
+ )
+
+
+@custumers_api.get("/custumers")
+def get_custumers():
+    return [{"id": custumer.id, "name": custumer.name, "cep": custumer.cep, "email": custumer.email} for custumer in Custumer.query.all()]
+
+
+@custumers_api.post("/custumers")
+def create_custumer(body: CustumersPostSchema):
+    print("#############", body)
```

Comment:
@PedroTeixeira-dev You can remove this print here 👍🏻

Reply:
done